cmd/serverless/main.go

```diff
@@ -128,7 +131,7 @@ func runAgent(tagger tagger.Component, compression logscompression.Component) {
 	wg.Add(3)
 
 	go startTraceAgent(&wg, lambdaSpanChan, coldStartSpanId, serverlessDaemon, tagger, rcService)
-	go startOtlpAgent(&wg, metricAgent, serverlessDaemon, tagger)
+	go startOtlpAgent(&wg, metricAgent, serverlessDaemon, tagger, gatewayUsage)
```

Comment:
Serverless could never be a gateway, so this isn't really required at all here. 

In general I feel like this metric should only be coming from the OTel Collector (aka `otel-agent`) but not necessarily other processes with some OTel processing capabilities like serverless or the `trace-agent`, etc. We should maybe discuss.

Reply:
Good to know. My understand was that it should be sent for serverless and OTLP.